flink-libraries/flink-gelly-examples/src/main/java/org/apache/flink/graph/drivers/parameter/LongParameter.java

```diff
@@ -52,16 +52,6 @@ public LongParameter(ParameterizedBase owner, String name) {
 	public LongParameter setDefaultValue(long defaultValue) {
 		super.setDefaultValue(defaultValue);
 
-		if (hasMinimumValue) {
-			Util.checkParameter(defaultValue >= minimumValue,
-				"Default value (" + defaultValue + ") must be greater than or equal to minimum (" + minimumValue + ")");
-		}
-
-		if (hasMaximumValue) {
-			Util.checkParameter(defaultValue <= maximumValue,
-				"Default value (" + defaultValue + ") must be less than or equal to maximum (" + maximumValue + ")");
-		}
-
```

Comment:
Right, `ExecutionConfig.DEFAULT_PARALLELISM` is currently `-1`. We're still checking the user configuration within the min and max bounds, it's only when no value is given that the default is used and now permitted to lie outside of the bounds.